easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'c-ares'
+version = '1.18.1'
+
+homepage = 'https://c-ares.org/'
+description = "c-ares is a C library for asynchronous DNS requests (including name resolves)"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'c-ares'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s-%s.tar.gz' % (name.replace('-', ''), version.replace('.', '_'))]
+checksums = ['414872549eec4e221b576693fdc9c9bce44ff794d0f1f06f2515b56a7f6ec9c9']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('Autotools', '20210726'),
+]
+
+preconfigopts = 'autoreconf -i -f &&'
```

Comment:
I see that this one is new compared to `c-ares-1.17.2-GCCcore-10.3.0.eb`. Any particular reason for it? Do you run into errors without the `autoreconf`?

Reply:
The sources from github only ship `configure.ac`. So this is needed to generate the `configure` script.

Reply:
Ok, pretty good reason then, thanks for letting me know ;-)